src/linagora.esn.contact/app/controllers/contact-list.controller.js

```diff
@@ -89,6 +91,27 @@ function ContactListController(
       });
   }
 
+  function toggleMenu() {
+    const contactAsidClass = angular.element(document.querySelector('.contact-aside'));
+    const contactMainContentElem = angular.element(document.querySelector('.contact-main-content'));
+
+    if (document.querySelector('.contact-aside').style.display === 'block' || !(document.querySelector('.contact-aside').style.display)) {
+      contactMainContentElem.removeClass('col-xl-10 col-md-9');
+      contactMainContentElem.addClass('col-xl-12 col-md-12');
+    } else {
+      contactMainContentElem.removeClass('col-xl-12 col-md-12');
+      contactMainContentElem.addClass('col-xl-10 col-md-9');
+    }
+    contactAsidClass.toggle('hidden');
+  }
+
+  function toggleSideMenuEvent() {
```

Comment:
I don't understand, why not use the same logic as the Calendar module ? Of course and it will be appreciate to refacto the Contact module and like this not do jquery code

Reply:
in calendar is more simple , there are already a service to toggle Menu . It is not the some case in contact SPA 
This is the calendar's function 
```   function toggleSideMenuEvent() {
      $rootScope.$on('toggleSideMenu', function() {
        const sidebarVisibility = calendarSidebarService.getSidebarVisibility();

        calendarSidebarService.setSidebarVisibility(!sidebarVisibility);
      });
    } 
````